integration_tests/test_math_02.py

```diff
@@ -1,7 +1,16 @@
-from math import (sin, cos, tan, pi, sqrt, log, log10, log2, erf, erfc, gamma,
+from math import (sin, cos, tan, pi, sqrt, cbrt, log, log10, log2, erf, erfc, gamma,
                   lgamma, asin, acos, atan, atan2, asinh, acosh, atanh,
                   tanh, sinh, cosh, hypot, copysign)
 
+from ltypes import i32, f64, i64
+
+pi: f64 = 3.141592653589793238462643383279502884197
+e: f64 = 2.718281828459045235360287471352662497757
+tau: f64 = 6.283185307179586
```

Comment:
Do we need this level of precision for testing? I think 4-5 decimal places is fine.

Reply:
Ultimately these constants should be imported from the builtin math module.